### PR TITLE
IPU:rmmod unsupported modules before doing in place upgrade

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID:2901
IPU:rmmod unsupported modules before doing in place upgrade from RHEL8 to RHEL9

### CODE REVIEW

**Comment by MiriamDeng** [comment]
@fbq815 @PaulYuuu can you help to review it ?

File: provider/in_place_upgrade_base.py [hunk 1]
```diff
@@ -130,6 +130,10 @@ def pre_upgrade_whitelist(self, test):
             error_info = self.params.get("error_info")
             if re.search(error_info, output):
                 pass
+            #rmmod unsupported modules since it's not supported on RHEL9
+            post_release = self.params.get("post_release")
+            if post_release == "release 9":
```
Comment (PaulYuuu) [inline provider/in_place_upgrade_base.py:135]: Can detect the guest name in cfg.
```
RHEL.9:
    rmmod_module = "rmmod ip_set;rmmod nft_compat"
```

Comment (PaulYuuu) [inline provider/in_place_upgrade_base.py:135]: As 9 is the psot release, so how about set this for RHEL8 in cfg
```
RHEL.8:
    rmmod_module = "rmmod ip_set;rmmod nft_compat"
```

Comment (MiriamDeng) [reply provider/in_place_upgrade_base.py:135]: Agreed, I have updated it and could you please help to review it again ? @PaulYuuu

Comment (PaulYuuu) [reply provider/in_place_upgrade_base.py:135]: Only need to check if `rmmod_module` is set, not need to check post release.

Comment (MiriamDeng) [reply provider/in_place_upgrade_base.py]: Updated, thanks

**Review by PaulYuuu — APPROVED** [review]

**Comment by PaulYuuu** [comment]
Remove invalid commit pls.

**Pull request merged**